15-node-js-express/.gitignore

```diff
@@ -0,0 +1,2 @@
+backend/node_modules
+backend/package-lock.json
```

Comment:
```suggestion
```
package-lock.json needs to be on the repository for managing the different library versions, it's only a file so it doesn't hurt any storage limit.